src/components/Game/Game.tsx

```diff
@@ -0,0 +1,283 @@
+import { useState, useEffect } from 'react'
+import Board from "../Board/Board";
+
+import "./style.css";
+import React from 'react';
+
+function Game() {
+    const [height] = useState(6)
+    const [width] = useState(6)
+    const [minesCount] = useState(8)
+    const [grid, setGrid] = useState<[GridCell[]]>([[]])
+    const [minesCards, setMinesCards] = useState<string[]>([])
+    const [ones, setOnes] = useState<string[]>([])
+    const [twos, setTwos] = useState<string[]>([])
+    const [threes, setThrees] = useState<string[]>([])
+    const [fours, setFours] = useState<string[]>([])
+    const [fives, setFives] = useState<string[]>([])
+    const [sixes, setSixes] = useState<string[]>([])
+    const [sevens, setSevens] = useState<string[]>([])
+    const [eights, setEights] = useState<string[]>([])
+    const [empties, setEmpties] = useState<string[]>([])
```

Comment:
Haluutko selittää lyhyesti miksi pelin/korttien tila on kuvattu tällä lailla? Tää näyttää aika kompleksilta omaan nenään. 😄

Reply:
Koska oon React_noob93 oli korttien asettaminen pelialueen vastaaville ruuduille tehty alun perin päälle liimaamalla ja keep it simple and stupid -taktiikalla, painotus enemmän stupidin puolella :D

Alun perin generointi oli Boardin renderin sisällä ja koko homma meni jokseenkin näin:
1) Generoidaan pelialue
-- siirrytään renderiin --
2) Generoidaan uudet täydet pakat
3) Käydään pelialueen ruudut läpi ja katsotaan mitä vastaavia kortteja tarvitaan (esim. miinan kohdalla haetaan seuraava vapaa miinaa vastaava kortti ja siirretään se miinakorttien taulukkoon)
4) Rakennetaan korttipohjainen pelialue siten, että haetaan ruutua vastaava kortti vastaavasta taulukosta ja siirretään se kyseisen taulukon perälle hienolla shift() + push() -kiepsautuksella
5) Kun korttipohjainen pelialue on rakennettu, ovat korttien taulukot taas 3)-kohdan jälkeisessä tilassa ja ne voidaan esittää pelialueiden alapuolella indikoimaan pelialueen ruutujen ja korttipohjaisen pelialueen ruutujen välistä yhteyttä - toisin sanoen, listataan mitkä kortit ovat pelissä ja mitä ruutua mikäkin kortti vastaa